src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectOptions.cs

```diff
@@ -106,6 +106,12 @@ public string DisplayName
         public string ClientSecret { get; set; }
 
         /// <summary>
+        /// Gets or sets if https is required for the metadata address or authority.
+        /// The default is true. This should only be disabled in development environments.
```

Comment:
Grammar police: `This should be disabled only in development environments.` :monkey_face: